src/app/workspace/organization/detail/payments-panel/payments-panel.component.ts

```diff
@@ -110,7 +110,7 @@ export class PaymentsPanelComponent implements OnInit, OnDestroy {
   private subscribeToActions() {
     this.paymentCreatedSubscription = this.actionsSubject.subscribe(data => {
       if (data.type === PaymentsActionType.CREATE_PAYMENT_SUCCESS) {
-        console.log("pajoment created");
+        console.log("payment created");
```

Comment:
I think @marvec did this on purpose. :)

Reply:
Yes, that was on purpose :-) To differentiate from another similar message I used during development. It does not matter. This is going to be removed anyway...